### ISSUE TITLE
Trouble installing with NextCloud AIO

### ISSUE THREAD

**Issue by wildernessfamily** [Blaok/nextcloud-inotifyscan#23]
Hi, I wanted to try out your script. I've been just using occ files:scan but, this seems to be much easier way with less overhead.
I have NextCloud AIO (All In One) installed, configured, and up and running on a Raspberry Pi 4 w/ 8GB.
I installed iNotifyScan per the README.md.w
www-data.ini file (removed comments in www-data.ini for this issue)
```
[DEFAULT]

interval = 1

[Instance1]

occ = occ

user = admin

docker = 33:nextcloud-aio-nextcloud

external_storage = no

php = php
```

When I run `sudo systemctl status nextcloud-inotifyscan@www-data it states
```
× nextcloud-inotifyscan@www-data.service - Nextcloud inotify scan daemon for www-data
     Loaded: loaded (/etc/systemd/system/nextcloud-inotifyscan@.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Mon 2023-07-03 14:26:22 MDT; 2s ago
    Process: 822859 ExecStart=/usr/local/bin/nextcloud-inotifyscan --config /etc/nextcloud-inotifyscan/www-data.ini (code=exited, status=1/FAILURE)
   Main PID: 822859 (code=exited, status=1/FAILURE)
        CPU: 448ms

Jul 03 14:26:22 nextcloud systemd[1]: nextcloud-inotifyscan@www-data.service: Main process exited, code=exited, status=1/FAILURE
Jul 03 14:26:22 nextcloud systemd[1]: nextcloud-inotifyscan@www-data.service: Failed with result 'exit-code'.
Jul 03 14:26:22 nextcloud systemd[1]: nextcloud-inotifyscan@www-data.service: Scheduled restart job, restart counter is at 5.
Jul 03 14:26:22 nextcloud systemd[1]: Stopped Nextcloud inotify scan daemon for www-data.
Jul 03 14:26:22 nextcloud systemd[1]: nextcloud-inotifyscan@www-data.service: Start request repeated too quickly.
Jul 03 14:26:22 nextcloud systemd[1]: nextcloud-inotifyscan@www-data.service: Failed with result 'exit-code'.
Jul 03 14:26:22 nextcloud systemd[1]: Failed to start Nextcloud inotify scan daemon for www-data.
```

I checked `journalctl -e -u swatchWATCH` and `journalctl -e` but did not show anything that could help. I also verified that python is installed (python 3.10.6) and that `python` (which is python3) is in `/usr/local/bin`.

I tried running nextcloud-inotifyscan from the command line `nextcloud-inotifyscan --config /etc/nextcloud-inotifyscan/www-data.ini` and it seems to be working. I do receive one warning when I call it:
```
INFO - watching /mnt/ncdata/admin/files <=> /admin/files
Couldn't watch /mnt/ncdata/admin/files: No such file or directory
```

Any guidance would be greatly appreciated, thanks!

FYI: just for testing, I did try changing in the `www-data.ini` `33:nextcloud-aio-nextcloud` to `www-data:nextcloud-aio-nextcloud` with the same results.

**Comment by Blaok** [comment]
Hi @wildernessfamily, does your `www-data` user have permissions to run `docker`? I suspect you'll want to run `nextcloud-inotifyscan` as a different user that can run `docker`, given the fact that running from command line seems to be working.

The `Couldn't watch /mnt/ncdata/admin/files: No such file or directory` error seems to be a bug when nextcloud is running with docker; the filesystem path being watched is reported by `occ` from within the docker container, which may be different on the outside. It looks like this can be fixed by parsing the output of `docker inspect`; I'll give it a try soon.

**Comment by Blaok** [comment]
The docker path bug should have been fixed by a742b42ca7ced11a079478441880264685a0a0c9 :)

**Comment by wildernessfamily** [comment]
@Blaok Thanks for getting back to me.
You were correct. `www-data` didn't have permission to run docker. As soon as I performed `sudo usermod -aG docker www-data` and restarted iNotifyScan it started up like a charm :).

Did a status check and it came back active
```
nextcloud-inotifyscan@www-data.service - Nextcloud inotify scan daemon for www-data
     Loaded: loaded (/etc/systemd/system/nextcloud-inotifyscan@.service; enabled; vendor preset: enabled)
     Active: active (running) since Thu 2023-07-27 14:03:04 MDT; 5s ago
   Main PID: 62273 (nextcloud-inoti)
      Tasks: 8 (limit: 9238)
     Memory: 19.1M
        CPU: 249ms
     CGroup: /system.slice/system-nextcloud\x2dinotifyscan.slice/nextcloud-inotifyscan@www-data.service
             ├─62273 /usr/bin/python /usr/local/bin/nextcloud-inotifyscan --config /etc/nextcloud-inotifyscan/www-data.ini
             └─62275 docker exec -u33 nextcloud-aio-nextcloud php occ config:system:get datadirectory

Jul 27 14:03:04 nextcloud systemd[1]: Started Nextcloud inotify scan daemon for www-data.
```
Thanks for putting your time into creating such an awesome script!
